ruamel-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: ruamel
+  version: 0.1.0
+
+source:
+  path: .
```

Comment:
The reason for this is that the "source code" is contained within the recipe.  That being the empty `ruamel/__init__.py` file, and the `setup.py` file.